endpointmanager/pkg/historypruning/historypruning.go

```diff
@@ -0,0 +1,98 @@
+package historypruning
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+
+	"github.com/lib/pq"
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/capabilityparser"
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/helpers"
+
+	"github.com/onc-healthit/lantern-back-end/endpointmanager/pkg/endpointmanager/postgresql"
+)
+
+// PruneInfoHistory checks info table and prunes any repetitive entries
+func PruneInfoHistory(ctx context.Context, store *postgresql.Store, threshold string, queryInterval string) {
+
+	var rows *sql.Rows
+	var err error
+
+	if len(queryInterval) != 0 {
+		rows, err = store.DB.Query("SELECT operation, url, capability_statement, entered_at, tls_version, mime_types, smart_response FROM fhir_endpoints_info_history WHERE (operation='U' OR operation='I') AND ((date_trunc('minute', entered_at) <= date_trunc('minute', current_date - interval '" + threshold + "' minute)) AND (date_trunc('minute', entered_at) >= date_trunc('minute', current_date - interval '" + queryInterval + "' minute))) ORDER BY url, entered_at ASC;")
+		helpers.FailOnError("", err)
+	} else {
+		rows, err = store.DB.Query("SELECT operation, url, capability_statement, entered_at, tls_version, mime_types, smart_response FROM fhir_endpoints_info_history WHERE (operation='U' OR operation='I') AND (date_trunc('minute', entered_at) <= date_trunc('minute', current_date - interval '" + threshold + "' minute)) ORDER BY url, entered_at ASC;")
```

Comment:
I think you could pull out the query string up to the threshold variable and reuse that for both the if and the else, and then just tack on their respective endings, which would make the line shorter! It would also be nice to add a comment here to say what data is being pulled from the database.